docker/default.conf

```diff
@@ -0,0 +1,102 @@
+map $http_accept_language $lang {
+  default '';
+  ~^zh zh;
+}
+
+server {
+  listen       80;
+  server_name  localhost;
+
+  if ( $host != 'www.jenkins.io' ) {
```

Comment:
this makes it super hard to test. Would it be better to check if host is jenkins.io?

Reply:
I totally agree with you this redirection should be done at the ingress level.